src/integrations/watchers/indexable-permalink-watcher.php

```diff
@@ -98,9 +104,9 @@ public function reset_permalinks() {
 			$this->indexable_helper->reset_permalink_indexables( 'term', $taxonomy );
 		}
 
-		$this->indexable_helper->reset_permalink_indexables( 'user' );
-		$this->indexable_helper->reset_permalink_indexables( 'date-archive' );
-		$this->indexable_helper->reset_permalink_indexables( 'system-page' );
+		$this->indexable_helper->reset_permalink_indexables( 'user', null );
+		$this->indexable_helper->reset_permalink_indexables( 'date-archive', null );
+		$this->indexable_helper->reset_permalink_indexables( 'system-page', null );
```

Comment:
Why are these added `null` values necessary?
As `public function reset_permalink_indexables( $type = null, $subtype = null, $reason = Indexing_Notification_Integration::REASON_PERMALINK_SETTINGS )` has `null` as the default for `$type` and `$subtype` already?